hawkular-wildfly-monitor/src/main/resources/module/main/module.xml

```diff
@@ -23,6 +23,7 @@
     <resource-root path="hawkular-bus-rest-client.jar"/> <!-- DELETE THIS ONCE WE NO LONGER SEND TWO MESSAGES -->
     <resource-root path="hawkular-dmr-client.jar"/>
     <resource-root path="hawkular-feed-comm-api.jar"/>
+    <resource-root path="hawkular-inventory-json-helper.jar"/>
     <resource-root path="hawkular-monitor.jar"/>
     <resource-root path="inventory-api.jar"/>
```

Comment:
based on the changes to the pom.xml, I assume this "inventory-api" was renamed so this should be removed from module.xml